dds-sweeper/dds-sweeper.c

```diff
@@ -873,9 +859,9 @@ void loop() {
         int parsed = sscanf(readstring, "%*s %u %u %u", &type, &_timing);
```

Comment:
For some reason, the extra `%u` in the format string is now causing my Pico to freeze when the `mode` command is used.

Reply:
That is odd. Not even sure why it's there. I'll remove it.